### PR TITLE
[RU-RU] Added and translated missing lines

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/resources/resources/assets/waila/lang/ru_ru.json [hunk 1]
```diff
@@ -1,115 +1,216 @@
...
-  "tooltip.waila.instrument.bell"             : "Колокольчики",
-  "tooltip.waila.instrument.guitar"           : "Гитара",
-  "tooltip.waila.instrument.chime"            : "Чаймс",
-  "tooltip.waila.instrument.xylophone"        : "Ксилофон",
-  "tooltip.waila.instrument.iron_xylophone"   : "Вибрафон",
-  "tooltip.waila.instrument.cow_bell"         : "Коровий колокольчик",
-  "tooltip.waila.instrument.didgeridoo"       : "Диджериду",
-  "tooltip.waila.instrument.bit"              : "Бит",
-  "tooltip.waila.instrument.banjo"            : "Банджо",
-  "tooltip.waila.instrument.pling"            : "Палочки",
```
Comment (deirn) [inline src/resources/resources/assets/waila/lang/ru_ru.json:31]: Missing translations:
```
  "tooltip.waila.instrument.zombie"                             : "Zombie",
  "tooltip.waila.instrument.skeleton"                           : "Skeleton",
  "tooltip.waila.instrument.creeper"                            : "Creeper",
  "tooltip.waila.instrument.dragon"                             : "Ender Dragon",
  "tooltip.waila.instrument.wither_skeleton"                    : "Wither Skeleton",
  "tooltip.waila.instrument.piglin"                             : "Piglin",
  "tooltip.waila.instrument.none"                               : "None",
```

File: src/resources/resources/assets/waila/lang/ru_ru.json [hunk 2]
```diff
@@ -1,115 +1,216 @@
...
+  "config.waila.plugin_minecraft.attribute.compact_desc"        : "Отображать очки жизней и брони цифрами а не иконками",
+  "config.waila.plugin_minecraft.attribute.icon_per_line"       : "Максимум инокон на строку",
+  "config.waila.plugin_minecraft.attribute.icon_per_line_desc"  : "Количество иконок жизни и брони в строке перед началом следующей",
+  "config.waila.plugin_minecraft.attribute.long_health_max"     : "Максимум очков жизни для отображения",
+  "config.waila.plugin_minecraft.attribute.long_health_max_desc": "Наибольшее число очков жизни для отображения в виде иконок, если у моба или игрока больше чем указано здесь, будут отображены цифры",
+  "config.waila.plugin_minecraft.attribute.long_armor_max"      : "Максимум очков брони для отображения",
+  "config.waila.plugin_minecraft.attribute.long_armor_max_desc" : "Наибольшее число очков брони для отображения в виде иконок, если у моба или игрока больше чем указано здесь, будут отображены цифры",
+
+  "config.waila.plugin_waila"                                   : "Ядро Waila",
+  "config.waila.plugin_waila.show_mod_name"                     : "Показывать названия модов",
```
Comment (deirn) [inline src/resources/resources/assets/waila/lang/ru_ru.json]: Missing translation:
```
  "config.waila.plugin_waila.show_item_mod_name"                : "Show Item Mod Names",
```

**Review by deirn — CHANGES_REQUESTED** [review]
You can ignore the formatting for now, I'll handle that myself.
There are a couple of missing translations from features unique to the master branch.

File: src/resources/resources/assets/waila/lang/ru_ru.json (hunk 1 above)
Comment (ghost) [reply src/resources/resources/assets/waila/lang/ru_ru.json:31]: Already added and translated these lines.

File: src/resources/resources/assets/waila/lang/ru_ru.json (hunk 2 above)
Comment (ghost) [reply src/resources/resources/assets/waila/lang/ru_ru.json]: Already added and translated these lines.

**Review by deirn — APPROVED** [review]

**Pull request merged**